mysite/scripts/snapshot_then_push.sh

```diff
@@ -30,16 +30,37 @@ cd "$PREFIX"
 
 ## Create a path to store the JSON in
 TEMPFILE="$(mktemp --suffix -snapshot.json.gz)"
+TEMPDB="$(mktemp --suffix -snapshot.db)"
+SNAPSHOT_SETTINGS_PATH="$(mktemp --suffix -snapshot_settings.py)"
 
 ## Do the slooow thing of snapshotting the database
 ./manage.py snapshot_public_data | gzip > "$TEMPFILE"
 
+## Create a new settings to specify the name of the temporary snapshot db
+echo "from mysite.settings import *; DATABASES = {
+    'default': {
+        'NAME': os.path.join(MEDIA_ROOT_BEFORE_STATIC, "$TEMPDB"),
+        'ENGINE': 'django.db.backends.sqlite3',
+        'CHARSET': 'utf8',
+    },
+}" > "$SNAPSHOT_SETTINGS_PATH"
+
+# Load in the snapshot data, which automagically generates the temporary db in root directory
+echo "TEMPFILE VARIABLE: " $TEMPFILE
+./manage.py syncdb --noinput --migrate --settings="$SNAPSHOT_SETTINGS_PATH"
+./manage.py migrate --settings="$SNAPSHOT_SETTINGS_PATH"
```

Comment:
This line is redundant with the line before it. @onceuponatimeforever can you submit a pull request to remove it, at some point?